doc/getting_started/installation_mac.rst

```diff
@@ -57,11 +57,11 @@ Install tools to build Zephyr binaries:
 
 .. code-block:: console
 
-   brew install cmake ninja dfu-util doxygen qemu dtc python3 gperf
-   curl -O 'https://bootstrap.pypa.io/get-pip.py'
-   ./get-pip.py
-   rm get-pip.py
+   brew install cmake ninja dfu-util doxygen qemu dtc python3 python2 gperf
```

Comment:
Please don't include python2. macOS ships with Python 2, no need to install another copy?

Reply:
I will then add my comments about Python present in my PR